cluster-scope/base/rbac.authorization.k8s.io/clusterrolebindings/kuberay-operator-leader-election/clusterrolebinding.yaml

```diff
@@ -0,0 +1,12 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
```

Comment:
Same as below, this is a namespaced resource, no need for it to be in the `cluster-scope`. It does not require elevated permissions to deploy.

Reply:
done